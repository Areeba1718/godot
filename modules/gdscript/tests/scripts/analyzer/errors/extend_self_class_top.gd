extends Self

func test():
	print('not ok')
