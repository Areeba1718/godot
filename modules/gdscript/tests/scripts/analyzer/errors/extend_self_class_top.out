GDTEST_ANALYZER_ERROR
Cannot extend "Self" in a non-nested class.
